client/v2/client.go

```diff
@@ -19,6 +21,10 @@ import (
 // could be travelling over the internet.
 const (
 	UDPPayloadSize = 512
+	MaxUDPPayload  = 60 * 1024
+
+	newlineString = "\n"
```

Comment:
this seems unecessary, if you're trying to optimize I think the compiler could deal with this for you